### DISCUSSION TITLE
How to Determine the X-ray Path Direction

### DISCUSSION THREAD

**Discussion by Olin1461** [astra-toolbox/astra-toolbox#479] [Q&A]
I have created the phantom with dimensions (num_slices, num_rows, num_cols) and generated the forward projection matrix to simulate cone-beam projection. However, I am not sure what the approximate X-ray path’s direction for view 0 would be. Is there any conventional direction for the projection?

Thanks!

**Comment by askorikov** [comment]
Hi! I believe for 0.0 angle, the projection will be in the direction of increasing row number in Python and in the direction of increasing column number in Matlab (because of different data layout).
